src/japi.c

```diff
@@ -239,6 +253,11 @@ int japi_register_request(japi_context* ctx, const char *req_name, japi_req_hand
 		return -4;
 	}
 
+	if(ctx->init && strncmp(req_name, bad_req_name, strlen(bad_req_name)) == 0) {
```

Comment:
The error message is a bit miss leading. May this should be seperated. --> if(ctx->init == false) {Error: Initialization neeeded before registering japi requests)

Reply:
The `ctx-> init` flag was only introduced to allow using this function during initialisation of "bad named" functions, so this is intended